src/envs/cluttered_table.py

```diff
@@ -267,3 +242,114 @@ def _any_intersection(pose: Array, radius: float,
             if distance <= (radius + other_radius):
                 return True
         return False
+
+    # Handle collision checking. We'll just threshold the angle between
```

Comment:
thanks for refactoring this into a new function. could you move this comment inside the function itself?